shared/business-rules/pizza-pricing-rule.ts

```diff
@@ -7,10 +7,12 @@ export default function (item: BasketItem) {
     const isWithBBQ_Base: boolean = item.options.indexOf("BBQ base") !== - 1;
     if (item.name === 'Free Choice') {
         price += Math.max(toppings.length - 5, 0) * PizzaToppingPrices[item.version];
-        price += isWithBBQ_Base ? 2 : 0;
+    } else if (item.name === "Potato Skins with your favourite topping") {
```

Comment:
This is the pizza pricing rule. Do you think it's the right place to put information about a side?